### PR TITLE
Apply Auth and template to goal doc

### PR DESCRIPTION
**Pull request by daliboz**
Add authentication do and apply the template to goal doc

### CODE REVIEW

File: content/millennium/dstu2/care-provision/goal.md [hunk 1]
```diff
@@ -32,24 +53,67 @@ Notes:
 
   - The `targetDate` parameter may be provided once with a prefix to imply a date range or without a prefix to search for goals at a specific date. Alternately it may be provided twice with `le`, `lt`, `ge`, or `gt` prefixes to search for goals within specific range. The date and prefix pairs must create a closed range.
```
Comment (MaxPhilips) [inline content/millennium/dstu2/care-provision/goal.md]: While we are here, change targetDate to targetdate

File: content/millennium/dstu2/care-provision/goal.md [hunk 2]
```diff
@@ -7,6 +7,23 @@ title: Goal | DSTU 2 API
 * TOC
 {:toc}
 
+## Overview
+
+The Goal resource describes intended objectives for a patient.  A Goal is typically expressed as a desired outcome or health state to be achieved by a patient over a period or specific point in time.  A Goal may address preventative health or mitigation of a diagnosis or problem.  Goals can be non-health care related (e.g. dance at a wedding).
```
Comment (MaxPhilips) [inline content/millennium/dstu2/care-provision/goal.md]: Minor nit: "achieved by a patient over a period or specific point in time." -> "achieved by a patient over a period or *at a* specific point in time." ?

**Comment by MaxPhilips** [comment]
+1

File: content/millennium/dstu2/care-provision/goal.md [hunk 3]
```diff
@@ -7,6 +7,23 @@ title: Goal | DSTU 2 API
...
+
+The following fields are returned if valued:
+
+* [id](http://hl7.org/fhir/dstu2/resource-definitions.html#Resource.id){:target="_blank"}
```
Comment (MaxPhilips) [inline content/millennium/dstu2/care-provision/goal.md]: Goal id

Comment (MaxPhilips) [reply content/millennium/dstu2/care-provision/goal.md]: Saw on the CarePlan PR that these are being switched to just Id

**Comment by daliboz** [comment]
Addressed comments

**Comment by dennispatterson** [comment]
+1

**Pull request merged**